### ISSUE TITLE
Display cards from NetrunnerDB deck in registration UI

### ISSUE THREAD

**Issue by haradan** [Project-NISEI/cobra#115]
Issue https://github.com/Project-NISEI/cobra/issues/114 will let the user choose a deck from their NetrunnerDB private decks. We would like this to show the list of cards in the deck once you've chosen it, as it will appear to the TO.

This should be shown before the user submits their deck registration. After they've submitted (in https://github.com/Project-NISEI/cobra/issues/116), they should still be able to see it as the TO sees it.

If they select a different deck from NRDB, the display of that deck should be separate. It would be nice to have an indication of what's different between the selected deck in NRDB and the stored deck.

It would be nice to link out to NetrunnerDB to be able to view/edit the deck there.

We can also review the display of the decks you've got selected now, in case we can make the UI cleaner or clearer once the decklist display is done.

### Decklist formatting

The decklist should look like the current form on the NSG site:

https://nullsignal.games/organizers/organizer-downloads/
https://nullsignal.games/wp-content/uploads/2022/09/Decklist-Sheet-2022.pdf

It doesn't have to be one to one copy, but the most important bits are the dual column setup, rows with alternating colour shade lines, and the number of cards & influence.

There's a separate issue for printing the decks: https://github.com/Project-NISEI/cobra/issues/155
